index.html

```diff
@@ -8434,6 +8434,11 @@ <h2>Edge Computing</h2>
               node with lower latency and optionally under more direct control of the user (improving
               privacy). This can be important for control applications (e.g. in robotics), computer
               graphics (e.g. gaming) and for applications processing imagery (e.g. facial recognition).</li>
+	    <li>For edge computing, "locality" is about network latency (and perhaps other properties,
+	      such as connection reliability or bandwidth) but there may also be requirements
+	      directly related to physical location, for example to satisfy regional privacy requirements.
+	      In such a case, the geolocation of both the edge computer and the device using it
+	      may be relevant.</li>

```

Comment:
```suggestion
	    <li>For edge computing, "locality" is about network latency (and perhaps other properties,
	      such as connection reliability or bandwidth), but there may also be requirements
	      directly related to physical location, such as to satisfy regional privacy requirements.
	      In such cases, the geolocation of both the edge computer and the device using it
	      may be relevant.</li>

```

Reply:
I personally think "for example" is clearer, keeping.  The other "In such cases" change is ok.